tests/reporter/kubernetes.go

```diff
@@ -70,10 +70,10 @@ const (
 )
 
 type KubernetesReporter struct {
-	failureCount int
-	artifactsDir string
-	maxFails     int
-	alwaysReport bool
+	failureCount      int
+	artifactsDir      string
+	maxFails          int
+	programmaticFocus bool
```

Comment:
My intention to _not_ capture the state of Ginkgo inside another variable was to separate the Why from the What. Because of this I chose `alwaysReport` because it states what we want to do and not why. That's why I would like to keep the name as is.

WDYT?

Reply:
If we had additional ways of enabling `alwaysReport` you would have a point. As is it's just a straight pass through of `report.SuiteHasProgrammaticFocus` at the moment the naming just makes it harder for the reader of the test output or even the `ReportSpec` code itself to understand why logs are being dumped.

Reply:
Maybe use `debugLogging` as name then? That would then better correspond to the log output message.

Reply:
While this remains a pass through of `report.SuiteHasProgrammaticFocus` I'd rather keep the variable and log message the same so it's clear to the reader of the code or log why we are dumping logs here.